library/Icingadb/Widget/HostListItemDetailed.php

```diff
@@ -32,8 +33,17 @@ protected function assembleFooter(BaseHtmlElement $footer)
         $statusIcons = new HtmlElement('div', Attributes::create(['class' => 'status-icons']));
 
         // ToDo(fs): Get `has_comments` from database
-        if ($this->item->comment->limit(1)->execute()->hasResult()) {
-            $statusIcons->addHtml(new Icon('comments', ['title' => t('This item has been commented')]));
+        if (! empty($this->item->state->last_comment->text)) {
```

Comment:
Why `->text`? Empty comments can't be a thing anyway.

Reply:
Because `..state->last-comment` is a `model/comment` instance with null values, but not null.

Reply:
Ah, I see. Then please check for `last_comment_id !== null`. A comment's text is nowhere the deciding factor if a comment is considered existent.